src/modules/blockchainApplication/explore/components/BlockchainApplicationAddSuccess/BlockchainApplicationAddSuccess.test.js

```diff
@@ -0,0 +1,26 @@
+import { fireEvent, screen } from '@testing-library/react';
+import { renderWithRouter } from 'src/utils/testHelpers';
+import BlockchainApplicationAddSuccess from './BlockchainApplicationAddSuccess';
+
+const props = {
+  history: {
+    push: jest.fn(),
+  },
+};
+
+describe('BlockchainApplicationAddSuccess', () => {
+  it('displays properly', () => {
+    renderWithRouter(BlockchainApplicationAddSuccess, props);
+
+    expect(screen.getByText('Perfect! Application has now been added')).toBeTruthy();
+    expect(screen.getByText('You can see a list of your applications on the dashboard.')).toBeTruthy();
```

Comment:
I think you can add a test to assert if the illustration image renders using `getByAltText`

Reply:
@eniolam1000752 The current assertions should suffice. Similar pages don't have such assertions